tests/network/user_defined_network/conftest.py

```
@@ -167,3 +169,18 @@ def udn_pod(
     ) as pod:
         pod.wait_for_status(status=Pod.Status.RUNNING)
         yield pod
+
+
+@pytest.fixture()
```

Ruff (0.16.2)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)